atc/exec/get_step.go

```diff
@@ -121,7 +118,7 @@ func (step *GetStep) Run(ctx context.Context, state RunState) error {
 
 	step.delegate.Initializing(logger)
 
-	variables := creds.NewVariables(step.secrets, step.metadata.TeamName, step.metadata.PipelineName)
+	variables := step.delegate.Variables()
```

Comment:
It's nice that we don't have to use the `step.metadata` for this anymore. :+1:

Reply:
Nice to hear that.